progressmonitor/__init__.py

```diff
@@ -244,7 +247,7 @@ def from_json(self, id, j):
         if 'TrackerName' in j.keys():
             t.with_name(j['TrackerName'], True)
         if 'EstimatedSeconds' in j.keys():
-            t.with_estimated_seconds(j['EstimatedSeconds'], True)
+            t.with_estimated_seconds(int(j['EstimatedSeconds']), True)
```

Comment:
This is coming back out of dynamo as a Decimal type. We could convert to float here instead, or just deal with Decimals. They don't play nice with arithmetic involving floats though